### ISSUE TITLE
train dino with human image for human Image retrieval got bad performance

### ISSUE THREAD

**Issue by guangdongliang** [facebookresearch/dino#203]
when train dino with human images which have faces, the similar retrieval performance is very bad. when I check f[acebook-demo-link](https://ssl-demos.metademolab.com/retrieval/openimages/RetrievalResults?query_image=https%3A%2F%2Fc6.staticflickr.com%2F8%2F7309%2F9134729667_7b2c016a3a_z.jpg) with faces like [img](https://img95.699pic.com/xsj/2n/fn/4s.jpg!/fh/300), the website said that "**Unable to process this image. Please make sure your image does not contain faces**". 

I wander that we can not train dino with big faces in images?

**Comment by woctezuma** [comment]
I think it is just to avoid people looking up other people.

![Picture](https://i.ibb.co/cQ2Kc9s/2022-06-18-23-38-59.png)

**Comment by guangdongliang** [comment]
@woctezuma thank you for replay!
Got it. the search result in the demo is of good performance. Does that ability come from dino model? if so, Is there any big training difference comparing to [imagenet](https://gimg2.baidu.com/image_search/src=http%3A%2F%2Fimg.mp.itc.cn%2Fq_70%2Cc_zoom%2Cw_640%2Fupload%2F20170627%2Ffd1333f3b59448bb8bb9c069efc2a194_th.jpg&refer=http%3A%2F%2Fimg.mp.itc.cn&app=2002&size=f9999,10000&q=a80&n=0&g=0n&fmt=auto?sec=1658182772&t=fac4f320eb22853f391d387c95dee267)? 

I got good performance for object and scenery. But it's bad if containing big face like img which I want to get similar cloth or facial feature. I am so confused.

Do you have any idea about that?

**Comment by woctezuma** [comment]
Yes, I believe the good performance comes from the DINO (**Di**stillation with **no** label). It seems that features extracted by self-supervision are good to describe semantic content.

Sorry, I cannot help with your issues with faces outside of the web demo. Maybe Mathilde will pop in to check the Github issues one day.

**Comment by mathildecaron31** [comment]
Hi @guangdongliang 
Have you managed to fix your issue ?

@woctezuma thank you so much for replying to the issues, you're doing a great job :D

**Comment by zdaiot** [comment]
I also found the same problem. I think it may be related to the use of blurring identifiable faces in DINOv2?
